build-files/abricate/1.2.0/Dockerfile

```diff
@@ -0,0 +1,109 @@
+  bioperl \
+  gzip \
+  unzip \
+  liblist-moreutils-perl \
+  libjson-perl \
+  libtext-csv-perl \
+  libfile-slurp-perl \
+  liblwp-protocol-https-perl \
+  libwww-perl \
+  libpath-tiny-perl \
+  git \
+  ncbi-blast+ \
+  wget && \
+  apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# get any2fasta
+RUN wget https://github.com/tseemann/any2fasta/archive/refs/tags/v${ANY2FASTA_VERSION}.tar.gz && \
+  tar -xvf v${ANY2FASTA_VERSION}.tar.gz && \
+  rm v${ANY2FASTA_VERSION}.tar.gz && \
+  cd /any2fasta-${ANY2FASTA_VERSION} && \
+  chmod +x any2fasta
+
+# download abricate; make /data; check dependencies
+RUN wget https://github.com/tseemann/abricate/archive/v${ABRICATE_VER}.tar.gz && \
+  tar -zxvf v${ABRICATE_VER}.tar.gz && \
+  rm -rf v${ABRICATE_VER}.tar.gz && \
+  mkdir /data
+
+# set $PATH
+# set perl locale settings for singularity compatibility
+ENV PATH="/abricate-${ABRICATE_VER}/bin:/any2fasta-${ANY2FASTA_VERSION}:$PATH"\
+  LC_ALL=C
+
+# set up default databases
+RUN abricate --setupdb
+
+# default command when running without commands
+CMD ["abricate", "--help"]
+
+# final working directory is /data
+WORKDIR /data
+
+# testing layer starts here
+FROM app AS test
+
+# Grab test Vcholerae genome, uncompress, and save to /data
+RUN wget -O - https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/018/083/565/GCA_018083565.2_PDT000793508.2/GCA_018083565.2_PDT000793508.2_genomic.fna.gz \
+  | gunzip -c > /data/GCA_018083565.2_PDT000793508.2_genomic.fna
+
+# to ensure this env variable is set in test layer
+ARG ABRICATE_VER
+
+# so that the below commands are run with /bin/bash shell and not /bin/sh - needed for bash-specific tricks below
+SHELL ["/bin/bash", "-c"]
+
+# tests shamelessly stolen and modified from: https://github.com/tseemann/abricate/blob/master/.github/workflows/ci.yml
+RUN set -x && \
+  cd /abricate-${ABRICATE_VER}/ && \
+  abricate --setupdb && \
+  abricate --version && \
+  abricate --help && \
+  abricate --check && \
+  abricate --list && \
+  ! abricate --doesnotexist || echo "fail expected" && \
+  ! abricate --threads 0 && \
+  ! (abricate test/assembly.fa | grep '~~~') && \
+  abricate test/assembly.fa > 1.tab && \
+  abricate test/assembly.fa.gz > 2.tab && \
+  abricate test/assembly.gbk > 3.tab && \
+  abricate test/assembly.gbk.gz > 4.tab && \
+  abricate --nopath test/assembly.gbk.gz | grep '^assembly.gbk.gz' && \
+  abricate --summary {1,2,3,4}.tab > summary.tab && \
+  abricate --summary 1.tab 2.tab 1.tab |& grep 'duplicate' && \
+  abricate --summary <(cat 1.tab 2.tab 3.tab) | wc -l | grep -w 4 && \
+  abricate test/assembly.txt |& grep ERROR && \
+  abricate not_exist.embl |& grep ERROR && \
+  abricate --threads `nproc` test/assembly.fa.bz2 | grep -i FOSFOMYCIN && \
+  abricate --threads `nproc` test/assembly.fa.bz2 | grep -i lactam && \
+  for DB in `abricate --list | cut -f1 | tail -n +2`; do abricate --db $DB test/assembly.fa > /dev/null ; done && \
```

Comment:
This is one of the slowest lines of the test, could you please update it to include the threads option? 
```
... abricate --threads `nproc` ...
```